src/core/services/gesture/gesture.js

```diff
@@ -643,7 +643,9 @@ function attachToDocument( $mdGesture, $$MdGestureHandler ) {
     updatePointerState(ev, pointer);
     pointer.endTime = +Date.now();
 
-    runHandlers('end', ev);
+    if (ev.type !== 'pointercancel') {
```

Comment:
Refer to our discussion in #10145

Pointercancel can be fired before pointerup. And that might mean that pointerup will not be fired. This is not good because a lot of cleanup and rollback logic often resides in the end event handler.

A fix of it might be browser detection based IMHO